.pre-commit-config.yaml

```
@@ -34,6 +34,7 @@ repos:
     hooks:
       - id: flake8
         additional_dependencies:
+        - flake8-no-pep420==2.3.0
```

The release is ready!

```suggestion
        - flake8-no-pep420==2.4.0
```

Finds these errors:

```
django-stubs/contrib/postgres/forms/jsonb.py:1:1: INP001 File is part of an implicit namespace package. Add an __init__.py?
django_stubs_ext/django_stubs_ext/db/models.py:1:1: INP001 File is part of an implicit namespace package. Add an __init__.py?
django_stubs_ext/setup.py:1:1: INP001 File is part of an implicit namespace package. Add an __init__.py?
django_stubs_ext/tests/test_aliases.py:1:1: INP001 File is part of an implicit namespace package. Add an __init__.py?
django_stubs_ext/tests/test_monkeypatching.py:1:1: INP001 File is part of an implicit namespace package. Add an __init__.py?
setup.py:1:1: INP001 File is part of an implicit namespace package. Add an __init__.py?
tests/test_error_handling.py:1:1: INP001 File is part of an implicit namespace package. Add an __init__.py?
```

`setup.py` can be ignored, but I think the others are worth adding the `__init__.py` files for.

---

>The release is ready!

so fast! 🔥